src/hostapi/wasapi/pa_win_wasapi.c

```diff
@@ -2974,6 +2982,22 @@ static PaError GetClosestFormat(IAudioClient *client, double sampleRate, const P
 
     if (hr == S_OK)
     {
+        // Workaround for the Realtek driver bug (see https://github.com/PortAudio/portaudio/issues/875):
+        // IAudioClient_IsFormatSupported() accepts Mono format (mistakenly) but fails in
+        // IAudioClient_Initialize(), this bug was probably fixed because it can not be reproduced
+        // with Realtek driver version 10.0.19041.3636, 19/10/2023, affects Output mode only
+        if (!output && isSharedMode && (params.channelCount == 1) && (sharedClosestMatch == NULL) &&
+            (deviceInfo->MixFormat.Format.nChannels > 1))
+        {
+            // Limit this workaround to Realtek devices only
+            if (strstr(baseDeviceInfo->name, "Realtek") != NULL)
+            {
+                // Force internal Mono to Stereo mixer
```

Comment:
I think this is less confusing:

```suggestion
                // Force our Mono to Stereo mixer mechanism
```